app/_how-tos/add-rate-limiting-tiers-with-kong-gateway.md

```diff
@@ -60,9 +60,11 @@ cleanup:
       include_content: cleanup/products/gateway
 ---
 
-## Steps
+### 1. Set up consumer authentication
 
-1. Add the following content to `kong.yaml` to enable the Key Authentication plugin. You need [authentication](/authentication/) to identify the consumer and apply rate limiting.
+We need to set up [authentication](/authentication/) to identify the consumer and apply rate limiting. In this guide, we'll be using the [Key Auth plugin](https://docs.konghq.com/hub/kong-inc/key-auth/) plugin, but you can use any [Kong authentication plugin](https://docs.konghq.com/hub/?category=authentication). 
```

Comment:
In our current style guide, the pronoun "we" is discouraged. Is that something we should revisit? On a personal note, I don't like it, but that's preference and I could be convinced otherwise 😄